code/game/jobs/job/security.dm

```diff
@@ -60,6 +60,61 @@
 	mechanical_difficulty = "Easy - Medium"
 	duties = "Manage all three Security branches. Keep track of potential and on-going threats (such as containment breaches). Work with other departments to respond to said threats."
 
+// IO  && IA Moved into a singular role
+/datum/job/tribunal_agent
+	title = "Internal Affair Agent"
+	department = "Command"
+	selection_color = "#2f2f7f"
+	department_flag = COM|SEC
+	total_positions = 3 // A rough estimate of how many IA you can get if you compacted them
+	spawn_positions = 3
+	supervisors = "Internal Tribunal Department Officer & Ethic Committee Liaison"
+	economic_power = 4
+	requirements = list(EXP_TYPE_COMMAND = 720, EXP_TYPE_SECURITY = 900, EXP_TYPE_BUR = 60)
+	minimal_player_age = 10
+	ideal_character_age = 27
+	alt_titles = list("Internal Affair Operative" = /decl/hierarchy/outfit/job/security/tribunal_operative)
+	outfit_type = /decl/hierarchy/outfit/job/security/tribunal_agent
+	class = CLASS_B
+	hud_icon = "hudbridgeofficer"
```

Comment:
```suggestion
	hud_icon = "hudtdia"
```
your newly added hud icon isnt being used!

Reply:
I added a new icon? I didn't remember doing that